docker-entrypoint.sh

```diff
@@ -93,11 +96,18 @@ fi
 
 # If we are using an Object Storage Bucket, we add a custom location file.
 # We also check if a file with the same name does not exist, to prevent the override.
-if [ -n "${NGINX_OSB_BUCKET}" ] && [ ! -f "/etc/nginx/conf.d/fragments/osb.conf" ]; then
+if [ -n "${NGINX_OSB_BUCKET}" ] && [ ! -f "/etc/nginx/conf.d/fragments/001-osb-default.conf" ]; then
   mkdir -p /etc/nginx/conf.d/fragments
-  # We add osb.conf to fragments if Nginx is configured to use a bucket.
+  # We add 001-osb-default.conf to fragments if Nginx is configured to use a bucket.
   # Env subst will be done later on all fragments files.
-  cp /templates/fragments/osb.conf /etc/nginx/conf.d/fragments/osb.conf
+  cp /templates/fragments/001-osb-default.conf /etc/nginx/conf.d/fragments/001-osb-default.conf
+
+  # If we want to activate the assets:// stream support over s3.
+  if [ "${NGINX_ASSETS_STREAM_OVER_S3}" = 1 ]; then
```

Comment:
I would add the `NGINX_ASSETS_STREAM_OVER_S3` variable with `0` as default in the definition section above.

Reply:
Do you mean to add this to the export section?